tools/create-release-branch.sh

```diff
@@ -197,7 +197,7 @@ fi
 # - Else if the prefix matches a plugin dir name, use that plugin's version.
 # - Else ask.
 if [[ "${PREFIXES[$PREFIX]}" != *$'\n'* ]]; then
-	BRANCH="$PREFIX/branch-${VERSIONS[0]%%-*}"
+	BRANCH="$PREFIX/branch-${VERSIONS[0]%%-beta}"
```

Comment:
At some point we may want to revisit this to decide if we want to also remove `-beta.2` and the like.